src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
-export interface Figure {
-
+function errorParams(...args: number[]): boolean {
+  return args.some((el: number): boolean => {
+    return el <= 0;
+  });
 }
 
-export class Triangle {
+type Form = 'triangle' | 'circle' | 'rectangle';
```

Comment:
Better use `enum` here - it will give you more control and flexibility for `Shape` type.